### PR TITLE
Refactored the entire repo

### PR DESCRIPTION
**Pull request by neelkirit**
Added meaningful routes
Added form
Standardized Naming Convention

### CODE REVIEW

File: src/components/body/DocTool/SMEPortal/SMEPortalHome/SMEPortalHome.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import React, {Component} from 'react';
+import '../../../RoutesApp/RoutesApp.css';
+import aruba from '../../../../../assets/aruba_logo.png';
+import HorizontalCard from '../../../../modules/HorizontalCard/HorizontalCard'
```
Comment (nabiharaza) [inline src/components/body/DocTool/SMEPortal/SMEPortalHome/SMEPortalHome.js:4]: So you haven't used the custom card we made?

Comment (neelkirit) [reply src/components/body/DocTool/SMEPortal/SMEPortalHome/SMEPortalHome.js:4]: Refactored the custom card name to VerticalCard and HorizontalCard, in `modules`

File: src/components/body/DocTool/TestDocumentForm/TestDocumentForm.jsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import React, {Component} from "react";
+import PageTitle from "../../../modules/PageTitle/PageTitle";
+import HorizontalCard from "../../../modules/HorizontalCard/HorizontalCard";
```
Comment (nabiharaza) [inline src/components/body/DocTool/TestDocumentForm/TestDocumentForm.jsx:3]: Is this import needed here?

File: src/components/body/Home/Home.js [hunk 3]
```diff
@@ -1,23 +1,44 @@
...
+        return (
+            <div className='container'>
+                <PageTitle title={'HTS Dashboard'} />
+
+                <div className='card'>
+                    <div className='card-body'>
+                        <div className='row'>
+                            <div className='col-sm-4'>
+                                <VerticalCard imgsrc={orange} title='DocTool' toplink='SME Portal' route_toplink = '/doc-tool/sme-portal/home'
+                                                    bottomlink='Engineer Portal' route_bottomlink='/'/>
```
Comment (nabiharaza) [inline src/components/body/Home/Home.js:25]: whats the route_toplink and route_bottomlink?

File: src/components/body/DocTool/TestDocumentForm/TestDocumentForm.jsx (hunk 2 above)
Comment (neelkirit) [reply src/components/body/DocTool/TestDocumentForm/TestDocumentForm.jsx:3]: Actually not, my bad, fixing it, good point  !

File: src/components/body/RoutesApp/RoutesApp.js [hunk 4]
```diff
@@ -1,23 +1,25 @@
 import React from 'react';
-import './routes_app.css';
-import Nav from '../../header/Nav';
+import './RoutesApp.css';
+import Header from '../../header/Header';
 import Login from "../../body/Login/Login";
-import SMEPortal from "../../body/SME-Portal/smePortal";
+import SMEPortalHome from "../DocTool/SMEPortal/SMEPortalHome/SMEPortalHome";
```
Comment (nabiharaza) [inline src/components/body/RoutesApp/RoutesApp.js:5]: 👍

File: src/components/header/Header.js [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+                    </li>
+                    <li className="nav-item">
+                        <a className="nav-link" href='/'>My Profile</a>
+                    </li>
+                    <li className="nav-item">
+                        <a className="nav-link" href='/SMEPortal'>Doc Tool</a>
+                    </li>
+                </ul>
+                <form className="form-inline my-2 my-lg-0">
+                    <input className="form-control mr-sm-2" type="search" placeholder="Search HTS" aria-label="Search" />
```
Comment (nabiharaza) [inline src/components/header/Header.js:31]: does this search work right now?

File: src/components/body/Home/Home.js (hunk 3 above)
Comment (neelkirit) [reply src/components/body/Home/Home.js:25]: So a bit of hard coding here.
The Vertical Card is taking 2 child URLs for the sake of space, actually it should take any no. of urls using a loop struct, but I hard coded 2 - toplink and bottomlink . I will ad a TODO here to make it a loop struct in future

File: src/components/header/Header.js [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+import '../body/RoutesApp/RoutesApp.css';
+
+function Header() {
+    const navStyle = {
+        color: 'white'
+    };
+    return (
+        <nav className="navbar navbar-expand-lg navbar-light bg-light">
+            <a className="navbar-brand" href="#">Aruba HTS</a>
+            <button className="navbar-toggler" type="button" data-toggle="collapse"
```
Comment (nabiharaza) [inline src/components/header/Header.js:12]: whats this button for?

File: src/components/header/Header.js (hunk 5 above)
Comment (neelkirit) [reply src/components/header/Header.js:31]: nopes 
dummy palceholder

File: src/components/header/Header.js (hunk 6 above)
Comment (neelkirit) [reply src/components/header/Header.js:12]: In mobile view, this is a hamburger, which will open the menu
so basically a menu collapses into this in smaller screens

File: src/components/header/Header.js [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+                    data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false"
+                    aria-label="Toggle navigation">
+                <span className="navbar-toggler-icon"></span>
+            </button>
+
+            <div className="collapse navbar-collapse" id="navbarSupportedContent">
```
Comment (nabiharaza) [inline src/components/header/Header.js:18]: Is this class 'collapse navbar-collapse' makes a the drop-down in the navBar. Could you please explain and make me understand?

File: src/components/modules/PageTitle/PageTitle.jsx [hunk 8]
```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import './PageTitle.css';
+
+const PageTitle = props => {
```
Comment (nabiharaza) [inline src/components/modules/PageTitle/PageTitle.jsx:4]: Do we make another file for even a thing such as page title?

File: src/components/modules/VerticalCard/VerticalCard.jsx [hunk 9]
```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import './VerticalCard.css';
+
+const VerticalCard = props => {
+    return (
+        <div className='row'>
+            <div className='col-sm-4'>
+                <div className="media">
+                    <img src={props.imgsrc} class="img-fluid" alt="image" width="200" height="200"/>
```
Comment (nabiharaza) [inline src/components/modules/VerticalCard/VerticalCard.jsx:9]: we could give the width and height of the image in css?

File: src/components/modules/VerticalCard/VerticalCard.jsx [hunk 10]
```diff
@@ -0,0 +1,23 @@
...
+                </div>
+            </div>
+            <div className='col-sm-8'>
+                <h3>{props.title}</h3>
+                <a href={props.route_toplink}>{props.toplink}</a>
```
Comment (nabiharaza) [inline src/components/modules/VerticalCard/VerticalCard.jsx:14]: same toplink, bottom link?

File: src/components/header/Header.js (hunk 7 above)
Comment (neelkirit) [reply src/components/header/Header.js:18]: no, its for collapsing in mobile view

File: src/components/modules/PageTitle/PageTitle.jsx (hunk 8 above)
Comment (neelkirit) [reply src/components/modules/PageTitle/PageTitle.jsx:4]: Ya `component`
The rationale behind this is, all page title looks same - standardization

File: src/components/modules/VerticalCard/VerticalCard.jsx (hunk 9 above)
Comment (neelkirit) [reply src/components/modules/VerticalCard/VerticalCard.jsx:9]: good point, 
actually it should be a `prop` will fix it in next PR
adding a TODO now

File: src/components/body/Home/Home.js (hunk 3 above)
Comment (nabiharaza) [reply src/components/body/Home/Home.js:25]: Nahi samnjha :/

File: src/components/modules/VerticalCard/VerticalCard.jsx (hunk 10 above)
Comment (neelkirit) [reply src/components/modules/VerticalCard/VerticalCard.jsx:14]: ya these accept value provided at the page where these components will be defined.

**Pull request merged**